tests/plugins/test_reactor_config.py

```diff
@@ -148,6 +149,115 @@ def test_get_docker_registry(self, config, fallback, valid):
                 with pytest.raises(OsbsValidationException):
                     get_docker_registry(workflow, docker_fallback)
 
+    @pytest.mark.parametrize(('config', 'expected'), [
+        ("""\
+            version: 1
+            pull_registries: []
+         """,
+         []),
+        ("""\
+            version: 1
+            pull_registries:
+            - url: registry.io
+         """,
+         [
+             {"uri": RegistryURI("registry.io"),
+              "insecure": False,
+              "dockercfg_path": None},
+         ]),
+        ("""\
+            version: 1
+            pull_registries:
+            - url: https://registry.io
```

Comment:
IIRC this is with API endpoint `https://registry.io/v2`

Reply:
Hmm, this like this in `registries` section, I see inconsistency. Ok, keep this consistent with source_registry